packages/fastfetch/build.sh

```diff
@@ -1,13 +1,12 @@
 TERMUX_PKG_HOMEPAGE=https://github.com/fastfetch-cli/fastfetch
-TERMUX_PKG_DESCRIPTION="A neofetch-like tool for fetching system information and displaying them in a pretty way"
+TERMUX_PKG_DESCRIPTION="A maintained, feature-rich and performance oriented, neofetch like system information tool"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="2.34.0"
+TERMUX_PKG_REVISION=2
```

Comment:
Package revisions on Termux start at `0` (implicitly) and the CI will ensure they are incremented sequentially, so this should be a `1`.
```suggestion
TERMUX_PKG_REVISION=1
```